app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+        sum(split_integer(32, 6)) == 32
+    ), "Sum of the split parts should be equal to initial value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (
+        len(set(split_integer(36, 6))) == 1
+    ), "Split parts should be equal when value divisible by parts"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert (
+        split_integer(32, 1)[0] == 32
+    ), "Split part should be equal to initial value if parts == 1"
```

Comment:
But what if there are more than 1 value in a result list? Your test should cover this case. Just compare functioin result to an expected list